### ISSUE TITLE
Remove the accessibility concern about the accordion using `aria-labelledby` on a `<div>` from the accessibility statement

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-design-system#3508]
## What

Once v5.1 has been released, remove the accessibility concern about the accordion using `aria-labelledby` from the accessibility statement:

> 1. The accordion component is currently using an aria-labelledby ARIA attribute on an unsupported <div> element. This is an incorrect implementation of [WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/), but is not a failure of WCAG success criterion [4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value). The team does not plan to fix this at present. Track our progress on the [GitHub issue: ‘Accordion - Elements must only use allowed ARIA attributes’](https://github.com/alphagov/govuk-frontend/issues/2472).

## Why

The linked issue has been resolved as part of v5.1.

## Who needs to work on this

Anyone

## Who needs to review this

Accessibility Specialist

## Done when

- [x] The accessibility statement no longer includes references to the issue described in https://github.com/alphagov/govuk-frontend/issues/2472
